django_test_migrations/signals.py

```diff
@@ -0,0 +1,13 @@
+from contextlib import contextmanager
+
+from django.db.models.signals import post_migrate, pre_migrate
+
+
+@contextmanager
+def mute_migrate_signals():
```

Comment:
Let's add docstring to avoid flake8 `D103` violation

Reply:
I wanted to, but couldn't figure out what the reason for muting the signals is. My test suite works fine without it, and I couldn't find a discussion of it. 
I'll add a stub and you are welcome to expand it.

Reply:
The only thing that comes to my mind is that some signals receivers/handlers could have some side effect that can potentially cause issues/crashes, so we are disabling it when preparing the testing database for the migration test